releaseNote.md

```diff
@@ -1,11 +1,7 @@
 ## Features
- - Upload build artifacts to blobstore (#3232)
- - Added ability to download artifacts from canceled builds. (#3254)
 
 ## Bugs
- - ResultsCommandExtension: Do not dispose the VssConnection (#3221)
- - Fix NullReferenceException when poolName is null (#3256)
- - Use node10 instead of node in runsvc.sh (#3267)
+ - Revert "Self-hosted agent cleaning source folder (#3237)"
```

Comment:
Since 2.183.0 is prelease and not fully rolled out, you want to leave all these other items in the release notes, because they will have been first released once 2.183.1 is released.